docs/usage/docker.md

```diff
@@ -206,6 +206,38 @@ module.exports = {
 };
 ```
 
+#### Google Container Registry
+
+Assume you are running GitLab CI in the Google Cloud, and you are storing your Docker images in the Google Container Registry (GCR).
+
+Access to the GCR uses a Bearer token based authentication. This token can be obtained by running `gcloud auth print-access-token`, which requires the Google Cloud SDK to be installed.
+
+It is also very important to note that this is a short-lived token ([60 minutes](https://stackoverflow.com/questions/50370714/google-cloud-bearer-token-expiry)) and thus storing it for subsequent builds in a variable (like you can do with the `RENOVATE_TOKEN`) is not an option.
+
+When running Renovate in this context the Google access token must be retrieved and injected into the `hostRules` configuration just before Renovate is started.
+
+_This documentation gives **a few hints** on **a possible way** to achieve this end result._
+
+The basic approach documented here is that you create a custom image and then run Renovate as one of the stages of your project. To make this run independent of any user you should use a `Project Access Token` for the project and use this as the `RENOVATE_TOKEN` variable for Gitlab CI. See also: https://gitlab.com/renovate-bot/renovate-runner
```

Comment:
```suggestion
The basic approach documented here is that you create a custom image and then run Renovate as one of the stages of your project.
To make this run independent of any user you should use a `Project Access Token` for the project and use this as the `RENOVATE_TOKEN` variable for Gitlab CI.
See also the [renovate-runner repository on GitLab](https://gitlab.com/renovate-bot/renovate-runner).
```

Use one sentence per line, and use a nice Markdown style link which explains where clicking on the link will take you.